### PR TITLE
test/k8sT: disable microscope test temporarily

### LINKED ISSUE #4965: CI: temporarily disable microscope test
#4879 is tracking a failure in the CI related to microscope not being able to run. The failure is related to microscope itself, not Cilium, and is being worked on actively by @nebril to be fixed on the microscope side. Disable the test since the failure is not related to Cilium functionality specifically.

### PR DESCRIPTION
**Pull request by ianvernon**
There is an issue on the microscope side that is causing CI to fail. This is
tracked by #4879, and is actively being fixed. This is purely related to
microscope, and not Cilium, so it does not make sense for Cilium CI to be
blocked by the microscope test failing. Temporarily disable the test.

Signed-off by: Ian Vernon <ian@cilium.io>

Fixes: #4965

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/4966)

### CODE REVIEW

**Comment by ianvernon** [comment]
test-me-please

**Review by eloycoto — APPROVED** [review]

**Review by nebril — CHANGES_REQUESTED** [review]
It seems like it was CRD-side error which Microscope was not prepared to handle, I am not that sure about disabling these tests now.

**Comment by tgraf** [comment]
@nebril Do we have another way of fixing it? All my CI runs have failed because of this.

**Comment by ianvernon** [comment]
After offline discussion we have decided to merge to unblock the pipeline. @nebril is still working on  fixing the issue.

**Pull request merged**
